README.rst

```diff
@@ -56,63 +56,63 @@ analysis tasks.
 Installation
 ============
 
-The latest release is available on PyPI and can be installed using
-``pip``:
+Renku releases and development versions are available from `PyPI
+<https://pypi.org/project/renku/>`_. You can install it using any tool that
+knows how to handle PyPI packages. Our recommendation is to use `:code:pipx
+<https://github.com/pipxproject/pipx>`_.
+
+
+``pipx``
+--------
+
+First, `install pipx <https://github.com/pipxproject/pipx#install-pipx>`_
+and make sure that the ``$PATH`` is correctly configured.
 
 ::
 
-    $ pip install renku
+    $ python3 -m pip install --user pipx
+    $ pipx ensurepath
 
-The latest development versions are available on PyPI or from the Git
-repository:
+Once ``pipx`` is installed use following command to install ``renku``.
 
 ::
 
-    $ pip install --pre renku
-    # - OR -
-    $ pip install -e git+https://github.com/SwissDataScienceCenter/renku-python.git#egg=renku
+    $ pipx install renku
+    $ which renku
+    ~/.local/bin/renku
 
-Use following installation steps based on your operating system and preferences
-if you would like to work with the command line interface and you do not need
-the Python library to be importable.
 
-Homebrew
---------
+``pipx`` installs renku into its own virtual environment, making sure that it
+does not pollute any other packages or versions that you may have already
+installed.
```

Comment:
I really like this paragraph and that's excatly what I meant by "we should improve the docs first". 😄 

Maybe we should add a small `NOTE:` section here which would describe that if user has installation in project virtual environment that that installation will be different from the pipx one and that upgrading in this case needs to be in accordance to the way renku is being used?

Reply:
I tried to add this using the `.. note:` directive but I'm not sure it's picked up by github...